spec/support/boxes/issue_box.cr

```diff
@@ -1,7 +1,7 @@
 class IssueBox < BaseBox
   def initialize
-    status Issue::AvramStatus.new(:opened)
-    role Issue::AvramRole.new(:issue)
+    status Issue::Status.new(:opened)
+    role Issue::Role.new(:issue)
```

Comment:
Could this be `Issue::Role::Issue`? This is totally fine as is Just curious if it is possible to do that with boxes

Reply:
Sadly no, as the constants are inside the enum, this is the wrapper. I haven't found a way to directly use enums.

Reply:
Would `Issue::AvramRole::Issue` work?

Reply:
Thinking about this, I don't think it would work. It would end up throwing an undefined method `parse` for that enum constant. I think my only hangup was that I wasn't too keen on the interface; however, I do see why it's like this.

We can probably just roll with this as is, and always change it up later if we come up with a new idea. Like `Issue::Status[:closed]` for example... This is a pretty decent start and definitely provides a lot more than what we have now. 👍